cfn-resources/cloud-backup-snapshot/README.md

```diff
@@ -0,0 +1,64 @@
+# MongoDB::Atlas::BackupSnapshot
```

Comment:
Update name to "MongoDB::Atlas::CloudBackupSnapshot" here and throughout the resource

Reply:
This issue is fixed